core-service/src/test/java/com/crmfoodestablishment/coreservice/service/OrderServiceImplTest.java

```diff
@@ -0,0 +1,52 @@
+package com.crmfoodestablishment.coreservice.service;
+
+import com.crmfoodestablishment.coreservice.dto.order.NewOrderDto;
+import com.crmfoodestablishment.coreservice.entity.Order;
+import com.crmfoodestablishment.coreservice.repository.DishRepository;
+import com.crmfoodestablishment.coreservice.repository.OrderRepository;
+import com.crmfoodestablishment.coreservice.service.exception.NotFoundException;
+import com.crmfoodestablishment.coreservice.service.impl.OrderServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
```

Comment:
don't use wild imports. This comment is relevant to all places

Reply:
This test has been removed.